blocks/content-intro/content-into.js

```diff
@@ -0,0 +1,41 @@
+export default function decorate(block) {
+  // Get the parent div element
+  const parentDiv = block.querySelector('div');
+
+  // Get the first inner div element
+  const innerDiv1 = parentDiv.querySelector('div');
+
+  // Get the (p) and (h2)  tag
+  const paragraph1 = innerDiv1.querySelector('p:first-of-type');
+  const heading2 = innerDiv1.querySelector('h2');
+  const paragraph2 = innerDiv1.querySelector('p:last-of-type');
+
+  // create new divs instead of p and h2 tag
+
+  const titleDiv = document.createElement('div');
+  const contentWrapper = document.createElement('div');
+  const subTitle = document.createElement('div');
+  const desc = document.createElement('p');
+
+  // Set the content of the new divs
+  titleDiv.innerHTML = paragraph1.innerHTML;
+  subTitle.innerHTML = heading2.innerHTML;
+  desc.innerHTML = paragraph2.innerHTML;
+
+  // adding class to divs
+  titleDiv.classList.add('title');
+  contentWrapper.classList.add('content-wrapper');
+  subTitle.classList.add('sub-title');
+  desc.classList.add('description');
+
+  // Create a new div element
+  const finalComponent = document.createElement('div');
+
+  // Append the (p) and (h2) tag to the new div
+  contentWrapper.appendChild(subTitle);
+  contentWrapper.appendChild(desc);
+  finalComponent.appendChild(titleDiv);
+  finalComponent.appendChild(contentWrapper);
+
+  block.innerHTML = finalComponent.innerHTML;
```

Comment:
Not sure what's the purpose of this script. We don't need to manipulate the dom elements to display the content intro block with a side area. That can be handled in the CSS.

This block structure should output a markup that allows you to style in CSS.

![Screen Shot 2023-07-20 at 9 44 34 AM](https://github.com/hlxsites/merative2/assets/1815714/55bf1e8d-a411-4ba6-8875-a3ad12334e60)